init.lua

```diff
@@ -283,7 +287,7 @@ minetest.register_chatcommand("rank", {
 		if #param == 1 and param[1] == "list" then
 			return true, "Available Ranks: "..ranks.list_plaintext()
 		elseif #param == 2 then
-			if minetest.get_player_by_name(param[1]) then
+			if minetest.get_player_by_name(param[1]) or ranks.get_rank(param[1]) then
```

Comment:
We don't really need to check if `minetest.get_player_by_name` returns anything, `ranks.get_rank` does all the work for us at once.

Reply:
I've removed it.